packages/flutter/lib/src/material/tabs.dart

```diff
@@ -712,7 +712,6 @@ class _TabBarState<T> extends ScrollableState<TabBar<T>> implements TabBarSelect
     setState(() {
       _tabBarSize = tabBarSize;
       _tabWidths = tabWidths;
-      _updateScrollBehavior();
```

Comment:
It used to work because with the SizeObserver stuff I added the scrollTo() call to _updateScrollBehavior, to match the semantics of updateExtents(). But it's not clear to me why the updateExtents call is resetting the sc-- oh, I see. We're not passing it in! Try changing _updateScrollBehaviour to passi in the current scrollOffset as the scrollOffset argument, and then see if that means we can add this line in _layoutChanged back...